libnmstate/netapplier.py

```diff
@@ -145,3 +143,12 @@ def _net_state_contains_sriov_interface(net_state):
 def _verify_change(plugins, net_state):
     current_state = show_with_plugins(plugins)
     net_state.verify(current_state)
+
+
+def _get_ignored_interface_names(plugins):
+    ifaces = []
+    for plugin in plugins:
+        for iface_name in plugin.get_ignored_kernel_interface_names():
+            ifaces.append(iface_name)
```

Comment:
Please use a `set()` in case plugins are returning the same interface name.

Reply:
Done, thanks!